privacy_bot/analysis/utils.py

```diff
@@ -8,6 +8,9 @@
 
 
 class Dataset():
+    """
+    Primarily used to ease data analysis tasks (e.g. text classification)
+    """
     def __init__(self, data, names):
         self.data = data
         self.names = names
```

Comment:
Aren't `self.names` and `self.labels` redundant? They seem to contain exactly the same information.